widget/embedded/src/containers/ExpandedQuotes/ExpandedQuotes.tsx

```diff
@@ -0,0 +1,53 @@
+import type { PropTypes } from './ExpandedQuotes.types';
+
+import { i18n } from '@lingui/core';
+import { Header } from '@rango-dev/ui';
+import React, { useEffect, useState } from 'react';
+
+import { HeaderButtons } from '../../components/HeaderButtons';
+import { Quotes } from '../../components/Quotes';
+
+import { Container, Content } from './ExpandedQuotes.styles';
+
+export const TIME_TO_CLOSE_OPEN_EXPANDED = 100;
+
+export function ExpandedQuotes(props: PropTypes) {
+  const { fetch, loading, onClickOnQuote, onClickRefresh, isVisible } = props;
+  const [isDelayedVisible, setIsDelayedVisible] = useState(false);
+  const containerClass = isDelayedVisible ? '' : 'is-hidden';
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout | undefined;
+
+    if (isVisible) {
+      timeout = setTimeout(() => {
+        setIsDelayedVisible(true);
```

Comment:
Do we need `settimeout` here? I removed it and it still worked.

Reply:
Because sometimes when we return to the home page, the animation doesn't work